### PR TITLE
Verify redeem and lend functions for pendle

### PR DESCRIPTION
**Pull request by sourabhmarathe**

### CODE REVIEW

File: test/lender/Lender.sol [hunk 1]
```diff
@@ -174,11 +174,12 @@ contract Lender {
   /// @param u address of an underlying asset
   /// @param m maturity (timestamp) of the market
   /// @param a the amount of underlying tokens to lend
-  /// @param mb the minimum amount of zero-coupon tokens to return accounting for slippage
+  /// @param minimumBought the minimum amount of zero-coupon tokens to return accounting for slippage
   /// @param d the maximum timestamp at which the transaction can be executed
-  function lend(uint8 p, address u, uint256 m, uint256 a, uint256 mb, uint256 d) public returns (uint256) {
+  function lend(uint8 p, address u, uint256 m, uint256 a, uint256 minimumBought, uint256 d) public returns (uint256) {
```
Comment (robrobbins) [inline test/lender/Lender.sol]: can `minimumBought` just be `r` ?

don't confuse our bitching about Element's `return_` var with this.

it **IS** the min amount of ZCT to return right? i mean the comment says so.. i like `r`

File: test/redeemer/Redeemer.sol [hunk 2]
```diff
@@ -92,18 +92,19 @@ contract Redeemer {
...
+    // Get the balance of tokens to be redeemed by the user
     uint256 amount = token.balanceOf(illuminate);
 
+    // Transfer the user's tokens to the redeem contract
     Safe.transferFrom(token, illuminate, address(this), amount);
 
+    // Redeem the tokens from the pendle contract
     IPendle(pendleAddr).redeemAfterExpiry(i, u, m);
-
-    emit Redeem(p, u, m, amount);
```
Comment (robrobbins) [inline test/redeemer/Redeemer.sol:105]: why does this `redeem` not emit the event?

**Review by robrobbins — CHANGES_REQUESTED** [review]

File: test/lender/Lender.sol (hunk 1 above)
Comment (sourabhmarathe) [reply test/lender/Lender.sol]: Yes it is, this can be `r`.

File: test/redeemer/Redeemer.sol (hunk 2 above)
Comment (sourabhmarathe) [reply test/redeemer/Redeemer.sol:105]: That should not have been removed.

**Review by robrobbins — APPROVED** [review]
looks good. 

make sure if we have that whole `minBought` vs `r` anywhere else to unify them all. i remember that in a few places between `lender` and `redeemer`

**Pull request merged**